articles/virtual-machines/workloads/sap/deployment-guide.md

```diff
@@ -1053,6 +1054,7 @@ The new VM Extension for SAP uses a Managed Identity assigned to the VM to acces
 1. Make sure to use SAP Host Agent 7.21 PL 47 or higher.
 1. Make sure to uninstall the current version of the VM Extension for SAP. It is not supported to install both versions of the VM Extension for SAP on the same virtual machine.
 1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].
++  >[!Note]:Install the lastest Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher +
```

Comment:
Please don't use NOTE section. IMHO this reduces the readability. Changing the text as described above should be good enough.